.github/workflows/android.yml

```diff
@@ -23,8 +23,8 @@ jobs:
           - triplet: 'x86-android'
 
     env:
-      ANDROID_PLATFORM: 21
-      ANDROID_TARGET_PLATFORM: 29
+      ANDROID_PLATFORM: 24
```

Comment:
Why is that?

Reply:
@m-kuhn , I couldn't get signature v2 (a requirement when targeting 30) to apply without raising minimum to 24.